### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by anion155** [rschiang/atom-language-qml#1]
Store package settings files in the `settings/` directory instead of `scoped-properties/`

```
Package.loadSettings (/usr/share/atom/resources/app/src/package.js:550:9)
<unknown> (/usr/share/atom/resources/app/src/package.js:172:43)
```

**Comment by k4nar** [comment]
Fixed by fb99e6e.

**Comment by rschiang** [comment]
Just realized my notification on this repo was turned off accidentally. (；´Д`A
Thanks for the report! I'm closing off this issue now.
